projects/ui-policy-studio-angular/src/lib/components/flow-details-phase-step/gio-ps-flow-details-phase-step.harness.ts

```diff
@@ -44,6 +45,13 @@ export class GioPolicyStudioDetailsPhaseStepHarness extends ComponentHarness {
     return (await (await this.locatorForOptional(DivHarness.with({ selector: '.info__description' }))())?.getText()) ?? undefined;
   }
 
+  public async getInfoMessage(): Promise<string | undefined> {
+    const tooltip = await this.locatorForOptional(MatTooltipHarness.with({ selector: '.info__head__info' }))();
+    // Fixme: await tooltip.getTooltipText(); returns empty string. when it's in a condition. I haven't figured out how to get over it.
```

Comment:
```suggestion
    // FIXME: await tooltip.getTooltipText(); returns empty string. when it's in a condition. I haven't figured out how to get over it.
```
not sure it's properly scanned in camel case ?